upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -54,7 +54,12 @@ data:
           "ipam": {
               "assign_ipv4": "{{ not IsIPv6Only }}",
               "assign_ipv6": "{{ IsIPv6Only }}",
+              {{- if not IsIPv6Only }}
               "type": "calico-ipam"
+              {{- else }}
+              "type": "host-local",
+              "ranges": [[{ "subnet": "usePodCidr" }]]
+              {{- end }}
```

Comment:
It would be cleaner to invert this test and put the IPv6 block first.

Reply:
Sure.